chpl/chpl-service/src/main/java/gov/healthit/chpl/scheduler/presenter/SurveillanceReportCsvPresenter.java

```diff
@@ -105,7 +105,7 @@ public SurveillanceReportCsvPresenter(Environment env, CertificationCriterionSer
         if (!productDetailsUrl.endsWith("/")) {
             productDetailsUrl += "/";
         }
-        productDetailsUrl += "#/product/" + data.getId();
+        productDetailsUrl += "#/listing/" + data.getId();
```

Comment:
can we use the new listingDetailsUrl property?

Reply:
yes only thing while doing it was it was asking to change env property to be protected. Are we okay doing it?

Reply:
You probably don't need to make it protected... the compiler is asking for that because "env" is a private field in the superclass. You should be able to access it with a getter. That is done elsewhere in this file. `getEnv().getProperty("listingDetailsUrl")`